data/Glitched World/Overworld.json

```diff
@@ -226,10 +245,10 @@
         "hint": "Lake Hylia",
         "time_passes": true,
         "events": {
-            "Bonooru": "is_child and Ocarina"
+            "Pierre": "is_adult and can_play(Scarecrow_Song)"
         },
         "locations": {
-            "Pierre": "is_adult and Bonooru and not free_scarecrow",
+            "Song from Bonooru": "is_child and Ocarina",
```

Comment:
Not understanding the intent of the Scarecrow Song changes here. (Maybe shouldn't be in this PR?)

Quick thoughts: this changes the notion of when you get the Scarecrow Song to playing for Bonooru as a child, then you "unlock" Pierre by playing it as adult, which changes the logic for the `Scarecrow` alias to require Pierre or free_scarecrow... (Honestly I'd just have free_scarecrow give the world.state Pierre in ItemPool.py).